### ISSUE TITLE
After charging my watch values do not get sent to the watch

### ISSUE THREAD

**Issue by Acido69** [wagnefrede/xDrip--Tasker-Tizen-Watchface-Integration#5]
Hi,
I love your watchfaces but I have a small issue, which is maybe my fault. When I go to bed I charge my watch and my phone. In the morning the watch does not get the actual glucose values from XDrip anymore. In Tasker I can see the correct values. Sending them manually to the watchface does not work. I have to remove the watchface from my watch and install it again. After this everything works fine until the next morning.

I am using a Samsung Galaxy s8+ in combination with a Samsung Galaxy S3 Frontier watch. Do you have any ideas?

Thanks in advance and please keep up your work on those excellent watchfaces.

**Comment by Acido69** [comment]
Quick update: This night nothing happened. The watchface ist still working. I haven't changed anything, maybe just a glitch on my side. I'll keep it under investigation and keep you informed.

**Comment by FreDiabetics** [comment]
Just open the watchmaker app on the phone one time and on the Watchface click the button to manual update glucose. After 1 sec everything should be actual again. Every of my Watchfaces have a button to execute the connected tasker tasks. Sometimes the Watchmaker companion background tasks on the watch stops working. I  have tried so much before I have contacted the devs for support nothing sorks.

I tried pi him the watchface every minute to send a Variabel back to tasker, to execute the tasks but even this has stopped working after X hours. 

This is sadly a known issue we are working on it's not related to my project it's a Tizen-Watchmaker Compan>n made issue I can't solve. This is not on my side. It's on dev side. The watchmaker devs are working to solve this ;)
